package.json

```diff
@@ -14,11 +14,14 @@
     "test": "yarn unit-test && yarn integration-test",
     "create-views": "knex seed:run --env views",
     "migrate-app": "knex migrate:latest --env app && node ./delete-views && yarn create-views",
-    "seed-app": "node ./delete-data.js && knex seed:run --env app",
+    "seed-dev": "knex seed:run --env dev",
+    "seed-ref": "knex seed:run --env app",
+    "delete-dev-data": "node ./delete-data.js dev",
+    "delete-ref-data": "node ./delete-data.js ref",
     "rollback-app": "knex migrate:rollback --env app",
     "migrate-stg": "knex migrate:latest --env staging",
     "rollback-stg": "knex migrate:rollback --env staging",
-    "reset-db": "yarn migrate-stg && yarn migrate-app && yarn seed-app"
+    "reset-db": "yarn migrate-stg && yarn migrate-app && yarn delete-dev-data && yarn delete-ref-data && yarn seed-ref && yarn seed-dev"
```

Comment:
Might want to take `seed-dev` out of this - that way for integration we can run `reset-db` and `seed-dev` separately, but all other environments can run this command and get only the ref data.